.eslintrc.js

```diff
@@ -1,29 +1,33 @@
 module.exports = {
-  extends: [
-    'plugin:@typescript-eslint/recommended'
-  ],
+  extends: ['plugin:@typescript-eslint/recommended'],
```

Comment:
can you reset this file, it doesn't look like you changed anything

Reply:
Yep will do